src/e2e/resources/test.travis.properties

```diff
@@ -11,15 +11,6 @@ test.geckodriver.path=/home/travis/geckodriver
 test.firefox.path=
 test.timeout=5
 test.persistence.timeout=16
-test.admin.account=yourGoogleId
-test.admin.password=adminpassword
-test.instructor.account=teammates.coord
-test.instructor.password=anypassword
-test.student1.account=alice.tmms
-test.student1.password=anypassword
-test.student2.account=charlie.tmms
-test.student2.password=anypassword
-test.unreg.account=teammates.unreg
-test.unreg.password=anypassword
+test.student.email=alice.tmms@example.tmt
```

Comment:
This email could also be used to test emails to instructors if needed so maybe can just call it `test.email`?